Add statistics and project quota documentation. (#788)
Enalean/tuleap-documentation-en: languages/en/administration-guide/application-management/plugins/statistics.rst

Statistics
==========

This plugin is used to see the disk usage by the different services (git, document, frs...) and projects on a Tuleap instance.

Project Quota
-----------------------------------

> Project quota is part of the plugin statistics and use the same data. It allows you to put a limit on the disk usage of a project.

Reviewer: ```suggestion
Project quota is part of the plugin statistics and uses the same data. It allows you to set a limit on the disk usage of a project.
```

> If the limit quota is reached, push with :ref:`git_lfs`  will not be possible anymore.

Reviewer: ```suggestion
Pushing objects with :ref:`git_lfs` is not possible when you reach the limit quota.
```

Usage of the passive voice should be avoided.

https://developers.google.com/tech-writing/one/active-voice?hl=en

> For other usages, this is only for informational purposes.

Reviewer: ```suggestion
For other usages, nothing is blocked: the quota is only an information source that you can use to detect abuses and misuses.
```

> As an administrator, you can define quota for projects in the Statistics panel in administration. By default, this quota is defined at 5GB.

Reviewer: ```suggestion
As an administrator, you can define a quota for projects in the Statistics panel in administration. By default, this quota is defined at 5GB.
```

Reviewer: It would be nice to be explicit here: are we talking about site administrators or project administrators or something else?

Also which administration are we talking about?

It might not be obvious to the users.
